x-pack/plugins/kuery_autocomplete/index.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { resolve } from 'path';
+
+
+export const kueryAutocomplete = (kibana) => new kibana.Plugin({
+  id: 'kuery_autocomplete',
+  publicDir: resolve(__dirname, 'public'),
+  require: ['kibana', 'xpack_main'],
```

Comment:
this is unnecessary, this plugin doesn't actually require the `kibana` or `xpack_main` plugins AFAICT